js/pdf.js

```diff
@@ -213,11 +213,26 @@ export const builder = {
     }
     for(const subPageMap of origPageMap) {
       const pageNumList = Object.keys(subPageMap)
-      const embeddedPages = await new_pdf.embedPages(pageNumList.map(k => subPageMap[k]))
+      const bbox = {
+        left: this._collectValueOrPlaceholder(document.getElementById("pdf_margin_left")), 
+        right: window.book.unified_source.maxWidth-this._collectValueOrPlaceholder(document.getElementById("pdf_margin_right")),
+        top: window.book.unified_source.maxHeight-this._collectValueOrPlaceholder(document.getElementById("pdf_margin_top")),
+        bottom: this._collectValueOrPlaceholder(document.getElementById("pdf_margin_bottom"))
+      }
+      console.log("doing that embedded page stuff...")
+      const embeddedPages = await new_pdf.embedPages(pageNumList.map(k => subPageMap[k]), pageNumList.map(k => bbox))
       embeddedPages.forEach((p,i) => pageMap[pageNumList[i]] = p)
     }
     return [pageMap, sheets]
   },
+  _collectValueOrPlaceholder: function(el) {
```

Comment:
I should not have copied this function over like this -- stupid code duplication 
meh